sda-commons-dependencies/build.gradle

```diff
@@ -119,7 +120,8 @@ dependencies {
 
     // sda-commons-server-hibernate
     api "org.postgresql:postgresql:42.7.1"
-    api "org.flywaydb:flyway-core:10.7.2"
+    api "org.flywaydb:flyway-core:${flywayVersion}"
+    api "org.flywaydb:flyway-database-postgresql:${flywayVersion}"
```

Comment:
Afaik SI is also using DB2 in base services. Should we manage that as well?

Reply:
We can, then I would add that to the documentation as well. At the moment it's just documented to work with postgre.

Reply:
Let's just wait until we get a specific request for DB2. I hope, that you can pull in the Flyway DB2 (or other) module in addition to the Postgres module.

But regarding the docs: Maybe [this](https://github.com/SDA-SE/sda-dropwizard-commons/blob/ba635ebac3dd4d5885234a533d5c3ae07ff70095/docs/server-hibernate.md?plain=1#L13) is too specific. Could you add the flyway postgres module and remove the versions?